### PR TITLE
Take Away Challenge

### PR DESCRIPTION
**Pull request by festinalent3**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,4 @@
+TWILIO_ACCOUNT_SID = 'YOUR_SID'
```
Comment (danldb) [inline .env]: this whole file should be in your `.gitignore`

File: lib/menu.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+  def initialize
+  self.store(:pizza_deluxe, 15.00)
+  self.store(:salad, 10.00)
+  self.store(:hamburger, 12.00)
+  self.store(:chicken_tikka_masala, 15.50)
+  self.store(:curry, 14.50)
+  end
+
+end
```
Comment (danldb) [inline lib/menu.rb]: no need for a class here - this is just a Hash.

File: lib/message.rb [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  private
+
+  def twilio_send(message_body)
+    begin
+      @client = Twilio::REST::Client.new ENV['TWILIO_ACCOUNT_SID'], ENV['TWILIO_AUTH_TOKEN']
+      message = @client.account.messages.create(:body => message_body,
+      :to => ENV['CUSTOMER_NR'],
+      :from => ENV['TWILIO_NR'])
+    rescue Twilio::REST::RequestError => e
+      puts e.message
```
Comment (danldb) [inline lib/message.rb]: avoid `puts` - this is presentation logic, it has no place in the /lib.

File: lib/order.rb [hunk 4]
```diff
@@ -0,0 +1,25 @@
+class Order
+
+  def initialize
+    @list = {}
+  end
+
+  def add(dish, quantity)
+    @list.store(dish.to_sym, quantity)
```
Comment (danldb) [inline lib/order.rb]: what happens if they then add another of the same dish?
```ruby
order.add(:fish, 2)
order.add(:fish, 1)
```
I would expect to see 3 fish now.

File: spec/menu_spec.rb [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+
+describe Menu do
+  subject(:menu) { described_class.new }
+
+  it 'presents a list of dishes with prices' do
+    expect(menu).to include :pizza_deluxe
+    expect(menu[:pizza_deluxe]).to eq 15
+
+    expect(menu).to include :hamburger
+    expect(menu[:hamburger]).to eq 12
```
Comment (danldb) [inline spec/menu_spec.rb]: In unit tests, only 1 expectation per test.

**Comment by danldb** [comment]
I like how clean and concise your methods are.

Your code is very clear and readable - this is a really tough skill to develop!

Keep an eye out for edge cases.

Use DIP to reduce coupling between your classes.

Your should always be able to easily describe the one job of a class in a single sentence - try doing that in a comment at the top of each class.
